.babelrc

```diff
@@ -1,4 +1,10 @@
 {
-  "presets": ["es2015", "react"],
-  "plugins": ["transform-class-properties", "transform-object-rest-spread"]
+  "presets": [
+    "env",
```

Comment:
Ah ok. Well without any targets it acts the same as preset-es2015 other than also providing support for es2016/es2017. env includes the same dependencies in package.json